src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -236,6 +319,92 @@ public void onTraversalStart() {
     }
 
     /**
+     * Builds the exon collection given the values of user arguments.
+     *
+     * @throws UserException if there is some inconsistency in user arguments and inputs.
+     * @throws GATKException if there was any problem parsing the content of the exome file.
+     * @return never {@code null}.
+     */
+    private ExonCollection<? extends Locatable> resolveExonCollection() {
+        final ExonCollection<? extends Locatable> result;
+        if (exomeFile != null) {
+            result = resolveExonCollectionFromExomeFile();
+        } else if (hasIntervals()) {
+            final SAMSequenceDictionary sequenceDictionary = getBestAvailableSequenceDictionary();
+            if (sequenceDictionary == null) {
+                throw new UserException("missing reference sequence dictionary, as no input contains such a dictionary... please consider to provide fail-over dictionary using " +
+                        "a reference (e.g. --reference my_reference.fasta)");
```

Comment:
Should this tool perhaps override `requiresReference()` to indicate that a reference is required?